<?xml version="1.0"?>
<launch>

  <arg name="model" default="$(find snakebot)/urdf/snakebot.urdf"/>
  <arg name="gui" default="true" />
  <arg name="rvizconfig" default="$(find snakebot)/rviz/urdf.rviz" />

  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <param name="use_gui" value="$(arg gui)"/>


  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node name="snakebot_jointstate_publisher" pkg="snakebot" type="snakebot_jointstate_publisher"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

</launch>
 
